### DISCUSSION TITLE
Donation link in Add-On page

### DISCUSSION THREAD

**Discussion by KCStephen** [rern/rAudio#1545] [Feature Requests]
I appreciate the effort that has gone into rAudio and been having fun working with it.  Perhaps I've missed it, but don't see a place to make a donation for your time and effort.  How about a link on the Add-On page to be able to say thanks with some $$ from time to time.

**Comment by Roland-B** [comment]
I've asked @rern previously and he says he doesn't take donations but you can contribute to a charity on his behalf. Top man he is. 

So I gave to charity and posted screenshot.

Roland.

**Comment by KCStephen** [comment]
Thanks Roland
